libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/NavigatorMapper.kt

```diff
@@ -328,6 +334,67 @@ internal class NavigatorMapper {
                     .alertGeometry(alert.getAlertGeometry())
                     .build()
             }
+            RouteAlertType.KINCIDENT -> {
+                val incidentInfo = alert.incidentInfo!!
+                IncidentAlert.Builder(
+                    alert.beginCoordinate,
+                    alert.distance
+                )
+                    .alertGeometry(alert.getAlertGeometry())
+                    .incidentId(incidentInfo.id)
+                    .also {
+                        val incidentType = when (incidentInfo.type) {
+                            RouteAlertIncidentType.KACCIDENT ->
```

Comment:
import all of them to make more readable
`KACCIDENT -> INCIDENT_ACCIDENT`

Reply:
imo it's less readable. I don't know where's these `values` are locate